### PR TITLE
app-crypt/qca: remove unused patch

### PR DESCRIPTION
**Pull request by mm1ke**
some minor fixes for `app-crypt/qca`

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Review by a17r — CHANGES_REQUESTED** [review]
Danger. `UnusedInherits` gives false positives.

File: app-crypt/qca/qca-9999.ebuild [hunk 1]
```diff
@@ -3,14 +3,13 @@
...
 
-inherit cmake kde.org multibuild out-of-source-utils qmake-utils
+inherit cmake kde.org multibuild out-of-source-utils
 
 DESCRIPTION="Qt Cryptographic Architecture (QCA)"
 HOMEPAGE="https://userbase.kde.org/QCA"
 
 LICENSE="LGPL-2.1"
 SLOT="2"
-KEYWORDS=""
```
Comment (a17r) [inline app-crypt/qca/qca-9999.ebuild:13]: Unnecessary change.

**Comment by mm1ke** [comment]
@a17r 
i've removed the changes to the ebuild and only kept the removing of the patch. 
Sorry i've checked the ebuild how it should use `qmake-utils.eclass` but didn't notice the usage in:
```
			-DQCA_FEATURE_INSTALL_DIR="${EPREFIX}$(${MULTIBUILD_VARIANT}_get_mkspecsdir)/features"
			-DQCA_PLUGINS_INSTALL_DIR="${EPREFIX}$(${MULTIBUILD_VARIANT}_get_plugindir)"
```
